packages/theme-chalk/src/dropdown.scss

```diff
@@ -28,7 +33,7 @@
     }
 
     @include b(scrollbar__bar) {
-      z-index: #{$--dropdown-menu-index + 1};
+      z-index: calc(var(--el-dropdown-menu-index + 1));
```

Comment:
There is a styles error same like #2760 on here, it many be `calc(var(--el-dropdown-menu-index) + 1)`

Reply:
Thanks. I will fix it quickly.

Reply:
Do you want to commit for it as a finder?

Reply:
Of course, I can create a PR